action.yml

```
@@ -117,6 +117,15 @@ runs:
         sudo systemctl enable --now cri-docker.socket
       shell: bash
 
+    # NOTE: We apply a workaround as of version 3.0.1 by passing
+    #       --egress-selector-mode=disabled by default as not doing so following
+    #       modern versions of k3s has led to issues with `kubectl exec` and
+    #       `kubectl logs`.
+    #
+    #       For more details, see
```

```suggestion
    #       For more details, see
    #       https://github.com/k3s-io/k3s/issues/5633
```
#5633 is the root issue, so it's useful to link straight to it